page_permissions.ini

```diff
@@ -61,9 +61,15 @@
 
 ; intake / referral
 /referral-form = ALLOW User
+/referral-form/*/*/* = ALLOW User
```

Comment:
This could just be `/referral-form/**` which will get any number of route params 1 or more